### ISSUE TITLE
Accessible Authentication (3.3.7): Include each step in an authentication process

### ISSUE THREAD

**Issue by smhigley** [w3c/wcag#1358]
I'd like to start with a theoretical use case:
A web app allows users to log in with only a username and password, but provides the option to set up 2 factor authentication if desired, though all 2FA methods require a cognitive function test.

Based on the current wording, it seems like this would pass since there is at least one authentication method available (username/password). If this interpretation is correct, it would require users to choose between security and ease of access, creating an unequal experience for anyone this criterion is intended to help.

Would it be possible to modify the wording to the following so that if (for example) 2FA is available, there is at least one accessible 2FA method?:
> "For each available level of authentication (e.g., 1FA, 2FA…), there must be at least one authentication method that does not rely on a cognitive function test."

"Level of authentication" could also be something like "authentication step" or "authentication factor."

**Comment by AidanA11y** [comment]
I agree @smhigley .  The current wording seems to set up a situation where users (or website/app creators) need to choose between between security and ease of access, creating an unequal experience for anyone this criterion is intended to help. 

I also think if WCAG is perceived as at odds with security best practices, or becomes a reason to weaken security then this  may make accessibility in conflict with security, and I think security would be prioritized by organizations faced with that that choice.  

@patrickhlauke  raises related concerns in #1364

**Comment by alastc** [comment]
> Based on the current wording, it seems like this would pass since there is at least one authentication method available (username/password). If this interpretation is correct, it would require users to choose between security and ease of access, creating an unequal experience for anyone this criterion is intended to help.

That is a common scenario, my understanding from the COGA TF is that, given the 2nd factor would be a blocker, at least people affected would be able to get in. 
However, I certainly take the point that it is not a desirable outcome overall.

> Would it be possible to modify the wording to the following so that if (for example) 2FA is available, there is at least one accessible 2FA method?:

This was explicit in the wording previously, I can't remember how/why it was tweaked out. 

We do have this in the understanding:
> "If there is more than one step in the authentication process, such as with multi-factor authentication, all steps should comply with this success criterion. There should be a path through authentication that does not rely on cognitive function tests."

So that is the intent. If it isn't obvious enough (normatively), we can tweak it back in. E.g.

"For each step in an authentication process that relies on a cognitive function test, at least one other method must also be available that does not rely on a cognitive function test."
